src/components/header/_Header.scss

```diff
@@ -0,0 +1,49 @@
+@import url("https://fonts.googleapis.com/css2?family=Alfa+Slab+One&display=swap");
+
+/*Variables*/
+$font-logo: "Alfa Slab One", "Arial Black", cursive;
+$font-primry: "Manrope", Arial, sans-serif;
```

Comment:
Typo, it should be `$font-primary`

I suggest you to create the following files
src/scss/
  - _Fonts.scss // store all the variables regarding fonts here
  - _Palette.scss // store all the variables regarding colors here
  - _Mixins.scss // store all mixins / functions

Then, in any SCSS file, you can import like:
@import "src/scss/_Fonts"

Whenever you need to access a variable

Reply:
ok ok, i go to apply the solution

Reply:
Attended